Eat_and_Greet/src/comps/Textbox/style.tsx

```diff
@@ -4,8 +4,9 @@ const TextboxStyles = StyleSheet.create({
     container: {
       margin: 20,
       width: '90%',
-      backgroundColor: '#FFF2F2',
+      backgroundColor: '#FFFFF2',
```

Comment:
stick with original as that's what's on the figma. If you disagree please attach both colors so we can see this change.

Reply:
Changed it so it's easier to read what is in the textbox. Looks like this now so it is a bit more white and easier to read
<img width="185" alt="Screenshot 2023-10-12 at 7 22 20 PM" src="https://github.com/apandey16/Meat_and_Feat/assets/80131235/0f7aa717-5d09-4afb-93ba-b9f360a50ea9">

Reply:
Change it on the figma too then otherwise we wont remember when using this coloring later.